pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/RoundDecimalTransformFunction.java

```diff
@@ -90,18 +90,31 @@ public double[] transformToDoubleValuesSV(ValueBlock valueBlock) {
     double[] leftValues = _leftTransformFunction.transformToDoubleValuesSV(valueBlock);
     if (_fixedScale) {
       for (int i = 0; i < length; i++) {
-        _doubleValuesSV[i] = BigDecimal.valueOf(leftValues[i])
-            .setScale(_scale, RoundingMode.HALF_UP).doubleValue();
+        try {
+          _doubleValuesSV[i] = BigDecimal.valueOf(leftValues[i])
+                  .setScale(_scale, RoundingMode.HALF_UP).doubleValue();
```

Comment:
Please apply [Pinot Style](https://docs.pinot.apache.org/developers/developers-and-contributors/code-setup#set-up-ide) and reformat the changes